Runtime/Triangulator.cs

```diff
@@ -422,6 +422,14 @@ public void Execute()
     {
         private static readonly TUtils utils = default;
 
+        static readonly ProfilerMarker MarkerPreProcessInputStep = new($"{nameof(PreProcessInputStep)}");
```

Comment:
This performance improvement (which I doubt is measurable), as far as I know, is only relevant for the editor and doesn't impact the build. I don't believe adding these statics is justified, so let's keep the code simple.